src/qml/components/StorageSettings.qml

```diff
@@ -28,7 +38,11 @@ ColumnLayout {
         actionItem: ValueInput {
             parentState: pruneTargetSetting.state
             description: optionsModel.pruneSizeGB
-            onEditingFinished: optionsModel.pruneSizeGB = parseInt(text)
+            onEditingFinished: {
+                root.customStorage = true
```

Comment:
I think custom storage should only be true if the values don't match the other two options.